cellengine/utils/types.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
+import sys
+from typing import Dict, List
+
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+TailoringDict = TypedDict(
+    "TailoringDict",
+    {
+        "inserted": List[Dict[str, str]],
```

Comment:
You could take this one step further and make these TypedDicts also.

Reply:
:+1: